trailblazer/clients/slurm_api_client/slurm_api_client.py

```diff
@@ -31,3 +28,12 @@ def get_job(self, job_id: str) -> SlurmJobResponse:
         except ValidationError as e:
             LOG.error(f"Error deserializing job response: {e}")
             raise ResponseDeserializationError(e)
+
+    def cancel_job(self, job_id: str) -> None:
+        endpoint: str = f"{self.base_url}/slurm/v0.0.40/job/{job_id}"
```

Comment:
I don't understand why this version is fixed in the code (v0.0.40) and not retrieved from somewhere else or stored as a constant if needed.

Reply:
Good point, it should be in the base url 👍

Reply:
I completely agree with this comment, but I'll save it for another PR to limit the headaches in this one